metabase/metabase
# Add tests to showing most recent dashboard revisions

Closes https://github.com/metabase/metabase/issues/16559
More context: https://metaboat.slack.com/archives/C057T1QTB3L/p1691061483085529

### Description

Actually this issue is probably fixed by https://github.com/metabase/metabase/pull/17887, which was a few months after the issue was created.

It's hard to verify if this reproduction works for the version at the time of the report (Jun 2021) since the dashboard revision is still in the modal, but now it's in the dashboard sidebar instead. So we can't simply try to run this test against older version.

### How to verify

Modifying a dashboard should always show its latest revision. e.g.
- Rename
- Add/modify description
- Toggle auto-apply filters
- Add/remove cards
- Move to another collection


### Checklist
